packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+
+import { Modal, Button, Text } from '@rentpath/react-ui-core'
+
+@themed(/^MoreOptionsModal/, { pure: true })
+
+export default class MoreOptionsModal extends PureComponent {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    theme: PropTypes.object,
+    moreOptionsForm: PropTypes.node,
+    total: PropTypes.number,
+    filterTotal: PropTypes.number,
+    onClose: PropTypes.func,
+    onSubmit: PropTypes.func,
+    clearFilters: PropTypes.func,
+    restoreFilters: PropTypes.func,
+  }
+
+  static defaultProps = {
+    theme: {},
+    total: 0,
+    filtersTotal: 0,
+    clearFilters: () => {},
+    onSubmit: () => {},
+    onClose: () => {},
+    restoreFilters: () => {},
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isOpen: props.isOpen,
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isOpen !== this.props.isOpen) {
+      this.setState({ isOpen: nextProps.isOpen })
+    }
+  }
+
+  @autobind
+  getShowPropertyButton() {
+    const { filterTotal, total } = this.props
+
+    let count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = 'Show 0 Properties'
+
+    if (count > 1) {
+      count = this.formatNumberWithCommas(count)
+      propertyCount = `Show ${count} Properties`
+    } else if (count === 1) {
+      propertyCount = 'Show 1 Property'
+    }
+
+    return propertyCount
+  }
+
+  @autobind
+  getPropertyCount() {
+    const { filterTotal, total } = this.props
+    let count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = '0 Properties Found'
+
+    if (count > 1) {
+      count = this.formatNumberWithCommas(count)
+      propertyCount = `${count} Properties Found`
+    } else if (count === 1) {
+      propertyCount = '1 Property Found'
+    }
+
+    return propertyCount
+  }
+
+  @autobind
+  handleSubmit() {
+    const {
+      onSubmit,
+      onClose,
+    } = this.props
+
+    if (onSubmit) {
+      onSubmit()
+    }
+
+    if (onClose) {
+      onClose()
+    }
+
+    this.setState({ isOpen: false })
+  }
+
+  @autobind
+  handleCancel() {
+    const {
+      onClose,
+      restoreFilters,
+    } = this.props
+    restoreFilters()
+    onClose()
+  }
+
+  formatNumberWithCommas(number) {
+    return String(number).replace(/(.)(?=(\d{3})+$)/g, '$1,')
+  }
+
+  renderHeader() {
+    const {
+      onClose,
+      theme,
+    } = this.props
+
+    const propertyCount = this.getPropertyCount()
+
+    return (
+      <div className={theme.MoreOptionsModal_Header}>
+        <Text
+          className={theme.MoreOptionsModal_PropertyCount}
```

Comment:
We don't need this className here.  You can just do `. MoreOptionsModal_Header .Text { ... } ` as the css qualifier

Reply:
@tadjohnston ok will do